pinot-core/src/main/java/org/apache/pinot/core/segment/index/readers/text/LuceneTextIndexReader.java

```diff
@@ -60,7 +66,7 @@
    * @param column column name
    * @param segmentIndexDir segment index directory
    */
-  public LuceneTextIndexReader(String column, File segmentIndexDir) {
+  public LuceneTextIndexReader(String column, File segmentIndexDir, int numDocs) {
```

Comment:
Recommend making second argument `indexDir` to denote that it is top level dir

Reply:
done